src/main/java/com/google/codeu/data/Message.java

```diff
@@ -25,20 +25,28 @@
   private String user;
   private String text;
   private long timestamp;
+  private String recipient;
 
   /**
-   * Constructs a new {@link Message} posted by {@code user} with {@code text} content. Generates a
+   * Constructs a new {@link Message} posted by {@code user} with {@code text} content
+   * and sent to {@code recipient}. Generates a
```

Comment:
nit: move some of the next line comment to this line to make it look a little nicer

Reply:
changed comment to next line